### DISCUSSION TITLE
Solutions for Serving GPKG data

### DISCUSSION THREAD

**Discussion by techmavengeospatial** [realiii/fudgeo#68] [Show and tell]
Serve GPKG GeoPackage SQLite databases:

- OGC API TILES/XYZ   Raster Tiles and PBF Vector Tiles  
- OGC API Features
- Dynamic Vector Tiles OGC API Tiles/XYZ & OGC WMTS from GPKG Vector Features (Geo data server only)
- Dynamic Raster Tiles OGC API Tiles/XYZ & OGC WMTS from GPKG vector features  (android tile server only)

Windows, Linux and MacOSX
Geo Data Server
https://geodataserver.techmaven.net/

Android Tile Server
https://techmaven.net/portabletileserver/

**Comment by jlhumber** [comment]
Thanks for pointing us to this technology
